### ISSUE TITLE
feat: multi-isolates support in Serinus

### ISSUE THREAD

**Issue by francescovallone** [francescovallone/serinus#71]
**Description**

Serinus applications should be able to be run on multiple isolates to increase the performance and the vertical scalability offered by the framework.

**Requirements**

- [ ] Create the Orchestrator
- [ ] Define a base implementation of the new application
- [ ] Reduce as much as possible the breaking changes for the existing applications

**Comment by francescovallone** [comment]
Currently cannot be implemented in the framework unfortunately.

**Comment by II11II** [comment]
https://github.com/dart-lang/sdk/issues/56841

**Comment by francescovallone** [comment]
One way to share everything can be to use [Shared Map](https://pub.dev/packages/shared_map) when dealing with Multi Isolates Applications. A more in-depth analysis is needed

**Comment by II11II** [comment]
Can [event_bus](https://pub.dev/packages/event_bus) be applied?

**Comment by francescovallone** [comment]
I don't think this one is the right use case. We need something that allows us to share memory between isolates or at the very least something that will keep all the instances updated with the last value.
